include/oneapi/tbb/concurrent_queue.h

```diff
@@ -346,6 +395,40 @@ class concurrent_bounded_queue {
                                          sizeof(queue_representation_type));
     }
 
+      concurrent_bounded_queue& operator=( const concurrent_bounded_queue& other ){
```

Comment:
Actually, all of the comments I left for `concurrent_queue` are applicable for `concurrent_bounded_queue` as well.